holoviews/element/graphs.py

```diff
@@ -8,6 +8,12 @@
 from .chart import Points
 from .path import Path
 
+try:
+    from datashader.layout import LayoutAlgorithm as ds_layout
+except:
+    ds_layout = None
+
+
 class graph_redim(redim):
```

Comment:
Minor point but I would call this ``redim_graph`` instead.

Reply:
Not really part of this PR but I'm happy to fix it here if you want me to.

Reply:
Please do!

Reply:
Done.